src/util/parented_ptr.h

```diff
@@ -2,83 +2,88 @@
 #define UTIL_PARENTED_PTR_H
 
 #include <QPointer>
+
 #include "util/assert.h"
+#include "util/memory.h"
```

Comment:
Prefer to include \<memory\> if you don't nee std::make_unique(). This file will become obsolete when switching from C++11 to C++14.